### ISSUE TITLE
Downloading events from time period

### ISSUE THREAD

**Issue by damirabdul** [TeslaGov/gdelt#1]
Is there any opportunity to download events from time period? I found `downloadAllSince(LocalDateTime since)` method and `until` param into `GdeltMultipleDownloadsConfiguration`, but I didn't find how to set both since and until params.

**Comment by kevinmichaelchen** [comment]
Good suggestion. In #2, I added a `downloadAllBetween` method that will do as you suggested. I will publish it to Bintray shortly.

**Comment by kevinmichaelchen** [comment]
@damirabdul Just published version 0.0.7 to Bintray.

You can pull with Gradle
```groovy
compile 'com.teslagov.gdelt:gdelt-java-sdk:0.0.7'
```
or Maven
```xml
<dependency>
    <groupId>com.teslagov.gdelt</groupId>
    <artifactId>gdelt-java-sdk</artifactId>
    <version>0.0.7</version>
    <type>pom</type>
</dependency>
```
